frontend/pages/_app.tsx

```diff
@@ -55,14 +55,8 @@ const App: NextPage<AppProps> = ({ Component, pageProps, err }: ExtendedAppProps
       const url = "/api/csrf/set/";
       axiosWithoutAuth.get(url);
     }
+  }, []);
 
-    // Scroll to the top of the page whenever router.push() is used.
-    // (The next/Link component automatically handles page scrolling,
-    // but router.push() does not.)
-    const handle = () => window.scrollTo({ top: 0, left: 0, behavior: "smooth" });
-    router.events.on("routeChangeComplete", handle);
-    return () => router.events.off("routeChangeComplete", handle);
-  }, [router.events]);
```

Comment:
Unrelated to the conversion, but no longer needed.
https://nextjs.org/blog/next-10-1#router-methods-scroll-to-top